test/semantics/kinds02.f90

```diff
@@ -0,0 +1,57 @@
+! Copyright (c) 2018, NVIDIA CORPORATION.  All rights reserved.
+!
+! Licensed under the Apache License, Version 2.0 (the "License");
+! you may not use this file except in compliance with the License.
+! You may obtain a copy of the License at
+!
+!     http://www.apache.org/licenses/LICENSE-2.0
+!
+! Unless required by applicable law or agreed to in writing, software
+! distributed under the License is distributed on an "AS IS" BASIS,
+! WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+! See the License for the specific language governing permissions and
+! limitations under the License.
+
+!ERROR: Integer(kind=0) is not a supported type
```

Comment:
We have been using all caps for Fortran in error messages, so I think it should be "INTEGER(KIND=0) is not a supported type".

Reply:
That "Integer" is coming from a call to `EnumToString()` on the `ENUM_CLASS(TypeCategory)` in `fortran.h`.

Maybe I should capitalize those enumerators; that would be a big change that should probably be made independently.

Reply:
Another approach is like in `operator<<(std::ostream &, const IntrinsicTypeSpec &)` which uses `parser::ToUpperCaseLetters` on the result of `EnumToString`.

Reply:
Sure, that seems bulletproof; will do.